src/db/collections/metrics/mod.rs

```diff
@@ -17,8 +17,19 @@ pub struct SyncMetrics {
     pub time: DateTime<Utc>,
     pub milestone_index: MilestoneIndex,
     pub milestone_time: u64,
-    #[cfg(feature = "analytics")]
-    pub analytics_time: Option<u64>,
+    #[influxdb(tag)]
+    pub chronicle_version: String,
+    #[influxdb(tag)]
+    pub network_name: String,
```

Comment:
Should we remove the `network_name`?

I think I was over-ambitious with adding this `tag`, because I don't think we ever use it to really differentiate stuff in Grafana. The `chronicle_version` makes sense because then we can more easily compare sync times across versions.

Reply:
Fine with me 🤷